examples/arctic_basin_seasonal_cycle.jl

```diff
@@ -62,35 +94,56 @@ for (i, time) in enumerate(times)
     set!(Ql[i], tabulated_latent[i:i])
 end
 
+# We define flux functions that linearly interpolate the time series:
+
 @inline function linearly_interpolate_flux(i, j, grid, Tₛ, clock, model_fields, flux)
     t  = Time(clock.time)
     return flux[i, j, 1, t]
 end
 
+# For shortwave radiation, we account for surface albedo:
+
 @inline function linearly_interpolate_solar_flux(i, j, grid, Tₛ, clock, model_fields, flux)
     Q = linearly_interpolate_flux(i, j, grid, Tₛ, clock, model_fields, flux)
-    α = ifelse(Tₛ < -0.1, 0.75, 0.64)
+    α = ifelse(Tₛ < -0.1, 0.75, 0.64) # albedo depends on surface temperature
     return Q * (1 - α)
 end
 
-# NOTE: Semtner (1976) uses a wrong value for the Stefan-Boltzmann constant (roughly 2% higher) to match
-# the results of Maykut & Unterstainer (196(5)). We use the same value here for comparison purposes.
-σ = 5.67e-8 * 1.02 # Wrong value!!
+# NOTE: Semtner (1976) uses a wrong value for the Stefan-Boltzmann constant
+# (roughly 2% higher) to match the results of Maykut & Untersteiner (1965).
+# We use the same value here for comparison purposes:
+
+σ = 5.67e-8 * 1.02 # Wrong value, but used for comparison!
+
+# We create flux functions for each component:
 
 Q_shortwave = FluxFunction(linearly_interpolate_solar_flux, parameters=Rs)
 Q_longwave  = FluxFunction(linearly_interpolate_flux,       parameters=Rl)
 Q_sensible  = FluxFunction(linearly_interpolate_flux,       parameters=Qs)
 Q_latent    = FluxFunction(linearly_interpolate_flux,       parameters=Ql)
 Q_emission  = RadiativeEmission(emissivity=ϵ, stefan_boltzmann_constant=σ)
```

Comment:
suggest coming up with mathematical notation for these (we will eventually codify this in Horizon/ClimaOcean)